.travis.yml

```diff
@@ -17,6 +17,7 @@ install: true
 before_script:
   - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
 script:
+  - set -e
```

Comment:
This required bash usage in travis which can change and we should not depends on it. More clear should be
```
mvn spotless:check -B -V || exit 1
```